app.py

```
@@ -54,6 +67,8 @@ def show_image(self):
             y = (screen_height / 2) - (popup_height / 2)
             popup.geometry(f"{popup_width}x{popup_height}+{int(x)}+{int(y)}")
 
+            logging.info(f"Popup window created at position ({x}, {y}) with size ({popup_width}x{popup_height}).")
```

Ruff

Logging statement uses f-string

(G004)